src/wast-parser.cc

```diff
@@ -1732,50 +1732,139 @@ Result WastParser::ParsePlainInstr(std::unique_ptr<Expr>* out_expr) {
   return Result::Ok;
 }
 
-// Current Simd const type is V128 const only.
-// The current expected V128 const lists is:
-// i32 0xXXXXXXXX 0xXXXXXXXX 0xXXXXXXXX 0xXXXXXXXX
-Result WastParser::ParseSimdConst(Const* const_,
-                                  Type in_type,
-                                  int32_t nSimdConstBytes) {
-  WABT_TRACE(ParseSimdConst);
-
-  // Parse the Simd Consts according to input data type.
-  switch (in_type) {
-    case Type::I32: {
-      const_->loc = GetLocation();
-      int Count = nSimdConstBytes / sizeof(uint32_t);
-      // Meet expected "i32" token. start parse 4 i32 consts
-      for (int i = 0; i < Count; i++) {
-        Location loc = GetLocation();
+  Result WastParser::ParseSimdV128Const(Const* const_, TokenType token_type) {
```

Comment:
nit: indentation looks off here

Reply:
Indeed. Would it be possible to add a `.clang-format` file or similar to the repo (and use a particular clang-format version on CI?). Every line I change in this repo requires me to manually insert spaces to indent things because my IDE is set up for the styles of other projects I work on, and that makes it painful to contribute here. Not other projects I work on follow the same style that my ide is set up in, but all of them use clang format, and my IDE is also set up to run clang-format on save, which is also something I have to disable / re-enable every time I switch to working on this project.

Reply:
Ok I decided to look and for some reason the repo does have a `.clang-format` file, but running clang-format generates a huge diff :/

Reply:
There does not seem to be a `make format` command that does the right thing, but AFAIK my IDE just invokes clang-format on the file, and clang-format recursively search parent directories for its configuration file, so I suppose the right file is being picked. I suspect that it just hasn't been invoked on the whole repo for a long time because it does not seem to be an easy way to do so here (no `make format` or `make format-check`). It might also be that the `clang-format` binary that my IDE picks is different than the one the repo is formatted with, but the diffs are so big that I doubt that. Usually `make format` is in charge of picking the right clang-format version (e.g. `clang-format-5.0` or `clang-format-8.0`), I have a couple of them installed, but I suspect that `clang-format` is the one that my IDE picks (which should be `clang-format-8.0`).

Reply:
Have you tried using `git clang-format`? That will only reformat lines that were touched in the commits you specify.

Reply:
I've used it in other projects, but i haven't tried to use it here. If that's the preferred way a note in the readme / contributing docs would be helpful. I think that just picking a particular clang-format version, and enforcing it on CI, would be easier though.